### DISCUSSION TITLE
Can we use ArabertPreprocessor for another multilingual BERT model?

### DISCUSSION THREAD

**Discussion by faiz-hub** [aub-mind/arabert#88] [Q&A] [answered]
Dear NLP Expert,
Can we just use ArabertPreprocessor.preprocess function only for preprocessing?, but for base model and tokenizer we use another pretrained BERT model (non Arabert).

Thank you in advance,
Faiz

**Comment by WissamAntoun** [comment, marked as answer]
yes, of course, just provide it with an empty string as a model_name and it should do fine

**Reply by faiz-hub** [reply]
Great, thank you @WissamAntoun
